.github/build.sh

```diff
@@ -1,3 +1,29 @@
 #!/bin/sh
+set -e
 curl -fsLO https://raw.githubusercontent.com/scijava/scijava-scripts/master/ci-build.sh
 sh ci-build.sh
+
+# Record the last time of cache modification before running melting-pot
+cache_modified_pre=0
+cache_modified_post=0
+
+if [ -f "~/.cache/scijava/melting-pot" ]; then
+  cache_modified_pre=$(find ~/.cache/scijava/melting-pot -printf '%T@\n' | sort -r | head -n 1)
+  cache_modified_pre=${cache_modified_pre%.*}
+fi
+
+# run melting-pot
+tests/run.sh
+
+# Record the last time of cache modification after running melting-pot
+if [ -f "~/.cache/scijava/melting-pot" ]; then
+  cache_modified_post=$(find ~/.cache/scijava/melting-pot -printf '%T@\n' | sort -r | head -n 1)
```

Comment:
@hinerm Heads up that BSD find (macOS) doesn't support `-printf` flag. Not necessarily a problem, since this script only runs on Linux right now, but we might want to future-proof this.

Reply:
Here is a version that works on macOS:
```
cache_modified_post=$(find ~/.cache/scijava/melting-pot/net.imagej/imagej | while read f
do
date -r "$f" +%Y-%m-%dT%H:%M:%S
done | sort -r | head -n1)

Reply:
@ctrueden incidentally that script seems to hang for me on Ubuntu so I'm just going to leave as-is for now, but I consolidated to a single method to make updating in the future easier.